frontend/src/slicer/orderSlice.ts

```diff
@@ -21,18 +22,24 @@ const initialState: IOrdersState = {
         items: []
     },
 }
-const validateOrder = (state: RootState) => {
-    const necessaryValues = ['name', 'suppliers', 'category', 'purchasePrice', 'unitSize']
-    const setValues = Object.keys(state.order.orderToSave)
-    return setValues.every(v => necessaryValues.indexOf(v) >= 0);
+const route = "orders_suppliers";
+export const validateOrder = (order: IOrder): boolean => {
+    if (!order.items.length || !order.supplier) return false;
+    return order.items.every(i => {
+        return i.product &&
+            i.product.suppliers?.some(s => s.id === order.supplier?.id)
```

Comment:
I wouldn't use those abbreviations here, as it is harder to read

Reply:
yeah I guess the laziness caught me here